packages/backend/src/appointment/appointment.handler.ts

```diff
@@ -0,0 +1,45 @@
+import { appointmentSchema } from "@carely/core";
+import {
+  insertAppointment,
+  getAppointmentsForElder,
+  deleteAppointment,
+} from "./appointment.entity";
+import { authenticated } from "../auth/guard";
+import z from "zod/v4";
+
+export const createAppointmentHandler = authenticated(async (req, res) => {
+  console.log("at createAppointmentHandler");
+  const appt = appointmentSchema.parse(req.body);
+  const newAppt = await insertAppointment(appt);
+  res.status(201).json(newAppt);
+});
+
+export const getAppointmentsHandler = authenticated(async (req, res) => {
+  console.log("at getAppointmentsHandler");
+
+  const { elder_id } = z
+    .object({
+      elder_id: z.string().transform((val) => parseInt(val, 10)),

```

Comment:
nit:

`z.coerce.number().int()` is preferred

or you could directly reference the elder schema

`elderSchema.shape.id`